src/main/datatypes/MQ2GroundType.cpp

```diff
@@ -377,4 +377,17 @@ bool MQ2GroundType::dataItemTarget(const char* szIndex, MQTypeVar& Ret)
 	return true;
 }
 
+MQTypeVar MQ2GroundType::MakeTypeVar(MQGroundSpawn groundSpawn)
+{
+	MQTypeVar Dest;
+
+	Dest.Type = pGroundType;
+	if (groundSpawn)
+		Dest.Set(groundSpawn);
+	else
+		Dest.Ptr = nullptr;
```

Comment:
Should remove the condition. We want this to be a null ground spawn, not a nullptr  when it doesn't exist.